x-pack/legacy/plugins/transform/public/app/services/http_service.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// service for interacting with the server
+
+import chrome from 'ui/chrome';
```

Comment:
Should you be using `npStart.core.chrome` here, or pulling this from the shim?

Reply:
Leaving as is for now, I tried migrating it but `chrome` via new platform doesn't have `chrome.getXsrfToken()` used in this file.